src/lib/y2partitioner/dialogs/partition_size.rb

```diff
@@ -1,6 +1,53 @@
+require "y2storage"
 require "yast"
 require "cwm/dialog"
-require "ui/greasemonkey"
+
+Yast.import "Popup"
+
+module Y2Storage
+  # Monkey-patched Region to get a size
```

Comment:
I guess the intention is to move all these monkey-patches (with its corresponding unit tests) to yast-storage-ng before closing the PBI. Isn't it? Isn't it? ISN'T IT? :wink:

Just asking because I don't see a FIXME/TODO here or a task in the PBI card